.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Tests on PR
+
+on:
+  workflow_dispatch:
+
+env:
+    API_KEY: ${{ secrets.API_KEY }}
+    APP_ID: ${{ secrets.APP_ID }}
+    APP_TOKEN: ${{ secrets.APP_TOKEN }}
```

Comment:
Is this the composer token?

Reply:
This makes more sense if I preface all of these with  `EXAMPLE_`. These env variables are set for the tests. 

However, I overlooked APP_TOKEN in my copy and paste. It doesn't need to be here.